src/features/search/componets/SearchBar/SearchBar.tsx

```
@@ -57,48 +74,111 @@
+                    role: 'option',
+                  }}
+                />
+              );
+            case 'mcda':
+              return (
+                <SelectItem
+                  key={item.name}
+                  className={cn(style.listItem, style.createMcdaAnalysis)}
+                  item={{
+                    title:
+                      '✨' + i18n.t('search.mcda_create_analysis') + ` "${item.name}"`,
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)